extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -573,6 +578,25 @@ export class AmpAnalytics extends AMP.BaseElement {
           return request;
         });
   }
+/* TODO REMOVE THIS BEFORE CHECKING IN!
```

Comment:
I didn't add this, so suspect it's a conflict from rebasing or some such. Will investigate.